.vsts-ci.yml

```diff
@@ -62,6 +55,21 @@ variables:
       /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
       /p:dotnetbuilds-internal-container-read-token-base64=$(dotnetbuilds-internal-container-read-token-base64)
 
+- ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: VmrRef
+    value: $(System.PullRequest.TargetBranch)
+- ${{ else }}:
+  - name: VmrRef
```

Comment:
Please put a comment that we need some existing branch for internal builds but the resource won't be used

Reply:
with your change this won't be needed right

Reply:
Yeah, I guess we'll have to merge those changes together